rust_src/src/keymap.rs

```diff
@@ -773,4 +773,15 @@ pub fn apropos_internal(regexp: LispStringRef, predicate: LispObject) -> LispObj
     }
 }
 
+/// Return a list of keymaps for the minor modes of the current buffer.
+#[lisp_fn]
+pub fn current_minor_mode_maps() -> LispObject {
+    // maps is initialized to null to silence the borrow checker. If this is
+    // not done then it will not compile since maps is not initialized
+    let mut maps: *mut LispObject = ptr::null_mut();
+    unsafe {
+        let num_of_maps = current_minor_maps(ptr::null_mut(), &mut maps);
```

Comment:
Hi, sorry for not getting back to this in so long. I have been quite busy.

The reason why `maps` is `&mut maps` is because when building remacs without it, the compiler warns me that the expected type is `*mut *mut lisp::LispObject`. In other words, it requires a double pointer to be passed to the function. Simply passing `maps` will result in a compilation error when cargo builds the project. Is there any other way of passing a pointer to a pointer in rust or have I misunderstood something critical. Sorry if this is a dumb question, I am still quite new to rust and the emacs codebase as a whole. Thanks in advance!